insurance/README.md

```diff
@@ -118,12 +111,17 @@
 - **Prediction Column Name**: charges
 
 ### Alerts
-Add Feature Alerts 
+Add Feature Drift Alerts 
  - The datageneration script will be generating drift on the following features - age, sex, bmi, region. 
  - Suggest to configure a separate alert for each individual feature. 
- - Use a threshold between 0 to 1. generally advised 0.02 to 0.03 inclusive.
+ - Use a threshold between 0 to 1. generally advised 0.5 to 0.6 inclusive.
```

Comment:
If I pick all continuous & use KS, what is the default value?
If I pick all categorical & use Chi Squared, what is the default value?
Our recommendation about be in the ball part right?